app/decorators/controllers/account_requests_controller_decorator.rb

```diff
@@ -1,7 +1,6 @@
-# Play nice with Ruby 3 (and rubocop)
 # frozen_string_literal: true
 
-AccountRequestsController.class_eval do
+::AccountRequestsController.class_eval do # rubocop:disable Style/RedundantConstantBase
```

Comment:
Not a big issue, but I think you need `# rubocop:enable Style/RedundantConstantBase ` after the block to  resume rubocop checks. Otherwise, If any new code gets added, it won't be checked for that particular cop